app.rb

```diff
@@ -1,135 +1,34 @@
-require './book'
-require './person'
-require './rental'
-require './student'
-require './teacher'
+require './books_choices'
+require './people_choices'
+require './rental_choices'
```

Comment:
### Optional

- [ ] It would be nice if we change the `require` to `require_relative` since we are working with files in our current directory :man_technologist:

Reply:
Thank you for the suggestion